powerhome/playbook
# [PLAY-493] dialog kit animation

#### Screens

<img width="540" alt="Screen Shot 2023-01-12 at 10 44 12 AM" src="https://user-images.githubusercontent.com/73671109/212112891-6feae083-e918-4ea7-acff-366b598ea801.png">

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-493)

#### How to test this

Open a left or right full height dialog and ensure it comes in for the left or right and not the top

#### Checklist:
